library/SimplePie/Sanitize.php

```diff
@@ -145,6 +146,25 @@ public function pass_file_data($file_class = 'SimplePie_File', $timeout = 10, $u
 		}
 	}
 
+	public function rename_attributes($tags = array())
+	{
+		if ($tags)
+		{
+			if (is_array($tags))
+			{
+				$this->rename_attributes = $tags;
+			}
+			else
+			{
+				$this->rename_attributes = explode(',', $tags);
```

Comment:
Is there any need to support comma separated strings, rather than just arrays of strings? Apps can always do the explosion themselves.

Reply:
It is the same as:
https://github.com/simplepie/simplepie/blob/1aec297145a150f627e6ed82b8b084e0c933def8/library/SimplePie/Sanitize.php#L148-L165